src/emulator/config/include/config/config.h

```diff
@@ -43,10 +43,13 @@ struct Config {
     int icon_size = 64;
     bool archive_log = false;
     bool texture_cache = true;
-    int sys_button = 1;
-    int sys_lang = 1;
+    bool fps_limit = false;
+    bool wait_for_vsync = true;
     std::string background_image = {};
     float background_alpha = 0.300f;
+    int desired_fps = 60;
+    int sys_button = 1;
+    int sys_lang = 1;
```

Comment:
Revert the changes in `config.h` and instead move your new values underneath `discord_rich_presence` like you have it in `config.cpp`. `config.h` mimics the placement of `config.cpp` for consistency.